src/Service.Tests/DatabaseSchema-PostgreSql.sql

```diff
@@ -217,6 +218,14 @@ CREATE TABLE mappedbookmarks(
     bkname text NOT NULL
 );
 
+CREATE TABLE books_sold (
+  id int PRIMARY KEY,
+  book_name TEXT,
+  copies_sold INT DEFAULT 0,
+  last_sold_on TIMESTAMP,
```

Comment:
Is there no datetime2 equivalent for Postgres? Also, how about `rowversion` ?

Reply:
This here: https://www.postgresql.org/docs/9.1/datatype-datetime.html suggests the equivalent of datatime is timestamp(n).There might be. Although here, we are not actually concerned with the datatype. It can be time/integer/string etc. We are concerned only with a column computed based on another column.

RowVersion or an equivalent is not present in PgSql.